src/test/java/org/jabref/preferences/OpenOfficeSyncWhenCitingTest.java

```diff
@@ -0,0 +1,37 @@
+package org.jabref.preferences;
+import org.jabref.logic.openoffice.OpenOfficePreferences;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+public class OpenOfficeSyncWhenCitingTest {
+
+    private static boolean previousValue;
+    private final JabRefPreferences preferences = JabRefPreferences.getInstance();
```

Comment:
Thanks for thinking of a test, but you don't need one here as you are just testing that a variable is stored in the preferences which is not that critical.  In JabRef we focus on testing the important things, e.g. model/logic.

To sum up => Just remove the test and it's all good.

And there is a general side effect you have here with the preferences in test , by using the JabRefPreferences Instance Object you are modifying the real preferences values which are also stored/persisted in the registry etc.  Therefore, if you need to test something which depends on the preferencesm you need to mock them and "fake" the return values. 
See for example:
```
   importFormatPreferences = mock(ImportFormatPreferences.class, Answers.RETURNS_DEEP_STUBS);
        when(importFormatPreferences.getKeywordSeparator()).thenReturn(',');
```

Reply:
Thanks for your feedback! I went ahead and removed that test. Thanks for your advice, I wasn't sure what would be worth testing.